way_to_home/tests/unittests/utils/testvalidators.py

```diff
@@ -0,0 +1,265 @@
+        self.assertTrue(result)
+
+    def test_date_validator(self):
+        """Method that tests the date_validator function"""
+        invalid_date = 123
+        result = date_validator(date=invalid_date)
+        self.assertEqual(result, None)
+
+        invalid_date = '12323fd'
+        result = date_validator(date=invalid_date)
+        self.assertFalse(result)
+
+        test_date = ['20190212', '2019-02-12', '12-02-2019', '12022019', '02122019', '12/02/2019']
+
+        for date in test_date:
+            result = date_validator(date=date)
+            self.assertNotEqual(result, None)
+
+    def test_start_date_validator(self):
+        """Method that tests the start_date_validator function"""
+        invalid_date = 123
+        result = start_date_validator(start_date=invalid_date)
+        self.assertFalse(result)
+
+        invalid_date = '12323fd'
+        result = start_date_validator(start_date=invalid_date)
+        self.assertFalse(result)
+
+        test_date = ['20190212', '2019-02-12', '12-02-2019', '12022019', '02122019', '12/02/2019']
+
+        for date in test_date:
+            result = start_date_validator(start_date=date)
+            self.assertTrue(result)
+
+    def test_end_date_validator(self):
+        """Method that tests the end_date_validator function"""
+        invalid_date = {'end_date': 123,
+                        'start_date': '2019-02-12'}
+        result = end_date_validator(**invalid_date)
+        self.assertFalse(result)
+
+        invalid_date = {'end_date': '2019-01-12',
+                        'start_date': '2019-02-12'}
+        result = end_date_validator(**invalid_date)
+        self.assertFalse(result)
+
+        test_date = {'end_date': '2019-02-12',
+                     'start_date': '2019-01-12'}
+        result = end_date_validator(**test_date)
+        self.assertTrue(result)
+
+    def test_time_validator(self):
+        """Method that tests the time_validator function"""
+        invalid_time = 123
+        result = time_validator(time=invalid_time)
+        self.assertFalse(result)
+
+        invalid_time = '12323fd'
+        result = time_validator(time=invalid_time)
+        self.assertFalse(result)
+
+        test_time = '21:43:23'
+        result = time_validator(time=test_time)
+        self.assertTrue(result)
+
+    def test_week_day_validator(self):
+        """Method that tests the week_day_validator function"""
+        invalid_week_day = 'fgs'
+        result = week_day_validator(week_day=invalid_week_day)
+        self.assertFalse(result)
+
+        invalid_week_day = 15
+        result = week_day_validator(week_day=invalid_week_day)
+        self.assertFalse(result)
+
+        invalid_week_day = -15
+        result = week_day_validator(week_day=invalid_week_day)
+        self.assertFalse(result)
+
+        test_week_day = 5
+        result = week_day_validator(week_day=test_week_day)
+        self.assertTrue(result)
```

Comment:
I think it is better to use for loop in range (0, 6) to validate all suitable week days.